app/assets/stylesheets/application.scss

```diff
@@ -205,3 +205,11 @@ body {
   font-size: 15px;
   font-family: Roboto;
 }
+
+.subject-group-credits-met {
+  color: green !important;
```

Comment:
it's a bummer we have to use `!important`, but looks like it is needed to override the list item color